### PR TITLE
Fix jsnext:main field in package.json

### PR DESCRIPTION
**Pull request by danny-andrews**
Related literature:

http://2ality.com/2017/04/setting-up-multi-platform-packages.html#jsnextmain-the-precursor-of-module
https://github.com/rollup/rollup/wiki/pkg.module

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -9,9 +9,6 @@
     },
     "production": {
       "plugins": ["transform-es2015-modules-commonjs"]
-    },
```
Comment (mihirsoni) [inline .babelrc]: This will be still required to support older version.

Comment (danny-andrews) [reply .babelrc]: Removing this is the whole point of this PR. [jsnext:main](https://github.com/jsforum/jsforum/issues/5) points to code with ES6 modules, not node modules. Anyone relying on this is relying on incorrect functionality.

Comment (danny-andrews) [reply .babelrc]: I understand if you think this is deserving of a major version bump, but it's incorrect as it is.

Comment (mihirsoni) [reply .babelrc]: Make sense, I guess we will need to do it as part of major bump version as this will break for the users who are using as it is now.

Comment (danny-andrews) [reply .babelrc]: Sounds like a plan, man.

**Comment by b1f6c1c4** [comment]
Still no update on this? It'd be better to merge this to 'redux-form-material-ui@next' instead of master.

**Comment by danny-andrews** [comment]
I'm gonna close this since it's three years old, and redux-form doesn't seem to be in favor in the React world anymore.

**Pull request closed without merging**